editor/plugins/animation_library_editor.cpp

```diff
@@ -783,7 +783,7 @@ void AnimationLibraryEditor::update_tree() {
 			for (const uint64_t &lib_id : collapsed_lib_ids) {
 				Object *lib_obj = ObjectDB::get_instance(ObjectID(lib_id));
 				AnimationLibrary *cur_lib = Object::cast_to<AnimationLibrary>(lib_obj);
-				StringName M = mixer->get_animation_library_name(cur_lib);
+				StringName M = cur_lib == nullptr ? StringName(TTR("[Global]")) : mixer->get_animation_library_name(cur_lib);
```

Comment:
The only place `"[Global]"` is used in the code, is the suffix of the tree item. It is never used as a StringName.
```cpp
	for (const StringName &K : libs) {
		TreeItem *libitem = tree->create_item(root);
		libitem->set_text(0, K);
		if (K == StringName()) {
			libitem->set_suffix(0, TTR("[Global]"));
		} else {
			libitem->set_suffix(0, "");
		}
```
As we can see in this code snippet, in the case that a global library is used, the element `K` in `libs` is `StringName()` in this case, not `StringName(TTR("[Global]"))`

However, it makes sense that we don't want error spam, and we also don't want it to match the name of an existing animation library (unless the user names their library `[Global]`). So would it be better to continue in the null case?
```suggestion
				if (cur_lib == nullptr) {
					continue;
				}
				StringName M = mixer->get_animation_library_name(cur_lib);
```

Reply:
That works as well, changed.